controller/value_resolver.rst

```diff
@@ -169,6 +169,17 @@ In addition, some components, bridges and official bundles provide other value r
     user has a user class not matching the type-hinted class, an ``AccessDeniedException``
     is thrown by the resolver to prevent access to the controller.
 
+:class:`Symfony\\Component\\Security\\Http\\Controller\\SecurityTokenValueResolver`
+    Injects the object that represents the current logged in token if type-hinted
+    with ``TokenInterface`` or a class extending ``TokenInterface``.
```

Comment:
```suggestion
    with ``TokenInterface`` or a class extending it.
```
It's a detail but I think we can avoid the repetition

Reply:
I agree, thanks !